proxy.go

```diff
@@ -365,7 +365,7 @@ func (p *proxyObject) proxyGetOwnPropertyDescriptor(name Value) (Value, bool) {
 
 		if resultDesc.Writable == FLAG_TRUE && resultDesc.Configurable == FLAG_TRUE &&
 			resultDesc.Enumerable == FLAG_TRUE {
-			return resultDesc.Value, true
+			return resultDesc.jsDescriptor, true
```

Comment:
I am pretty sure this needs to be changed ... but nothing breaks with or without that change in the tc39 tests so I might be wrong ... didn't actually read the EcmaScript on Proxies and getOwnPropertyDescriptor :(

Reply:
This is not a bug. The current contract for getOwnProp*() is that it can return either *valueProperty or, for simple properties (i.e. writable, configurable and enumerable data properties), the Value itself.

In general, I think there is a better way of implementing Object.entries() which requires a bit of enumeration refactoring. I'm going to push it shortly.